packages/core/src/lib/button/button.svelte

```diff
@@ -41,37 +42,50 @@ export let icon = '';
 
 /** The width of the button. */
 export let width: 'full' | 'default' = 'default';
+
+/** Additional CSS classes to pass to the button. Can be a `classnames.Arguement`. */
+export let classes: ClassNames = '';
+$: classList = classNames(
```

Comment:
@mcous and here is how we are using the `class-names` helper. There may be a cleaner way to do this but not sure.